Assets/Scripts/UI/CopilotUI.cs

```diff
@@ -0,0 +1,145 @@
+    public GameObject feebeePrefab;
+    public GameObject daddyLongLegs;
+    #endregion
+    private void Awake()
+    {
+        if (singleton == null)
+            singleton = this;
+        else
+            Destroy(gameObject);
+    }
+    private void Start()
+    {
+        foreach (CopilotData copilot in ProfileManager.instance.GetCurrentCopilotList())
+        {
+            GameObject tempCopilotButton = Instantiate(copilotButtonPrefab);
+            CopilotInfo tempCopilotInfo = tempCopilotButton.GetComponent<CopilotButton>().copilotInfo;
+            switch (copilot.name)
+            {
+                case "Feebee": 
+                    feebeePrefab.GetComponent<CopilotInfo>().copilotData = copilot;
+                    tempCopilotInfo.CopyInfo(feebeePrefab.GetComponent<CopilotInfo>());
+                    break;
+                case "Daddy Long Legs": 
+                    daddyLongLegs.GetComponent<CopilotInfo>().copilotData = copilot;
+                    tempCopilotInfo.CopyInfo(daddyLongLegs.GetComponent<CopilotInfo>());
+                    break;
+            }
+            tempCopilotButton.GetComponent<CopilotButton>().SetButton();
+            tempCopilotButton.transform.SetParent(scrollViewContent.transform);
+            tempCopilotButton.transform.localScale = Vector3.one;
+        }
+        start.interactable = false;
+    }
+    public void CharacterSelected(CopilotInfo copilot)
+    {
+        selected = copilot;
+        if (!selectionPanel.activeSelf) selectionPanel.SetActive(true);
+        activeDescription.text = selected.activeDescription;
+        passiveDescription.text = selected.passiveDescription;
+        portrait.sprite = selected.portrait;
+        if (selected.passive==passive.text)
+        {
+            activeButton.interactable = false;
+        }
+        else activeButton.interactable = true;
+        if (selected.active==active.text)
+        {
+            passiveButton.interactable = false;
+        }
+        else passiveButton.interactable = true;
+
+    }
+    public void SelectPassive()
+    {
+        passive.text = selected.passive;
+        passiveIcon.sprite = selected.passiveIcon;
+        Copilots.singleton.passive = selected.passive;
+        activeButton.interactable = false;
+        passiveSelected = true;
+        if (activeSelected && passiveSelected) start.interactable = true;
+    }
+    public void SelectActive()
+    {
+        active.text = selected.active;
+        activeIcon.sprite = selected.activeIcon;
+        Copilots.singleton.active = selected.active;
+        passiveButton.interactable = false;
+        activeSelected = true;
+        if (activeSelected && passiveSelected) start.interactable = true;
+    }
+    public void StartGame()
+    {
+        SceneManager.LoadScene(sceneToLoad,LoadSceneMode.Single);
+    }
+    public void GoBackToHubPrompt()
+    {
+        goBackPrompt.SetActive(true);
+    }
+    public void GoBackToHubPromptConfirm()
```

Comment:
Add specific comment on that this is when the player has confirmed they want to return to the hub